crates/rust-analyzer/src/main_loop.rs

```diff
@@ -486,6 +488,27 @@ impl GlobalState {
             // Only retry requests that haven't been cancelled. Otherwise we do unnecessary work.
             Task::Retry(req) if !self.is_completed(&req) => self.on_request(req),
             Task::Retry(_) => (),
+            Task::FileIndexState(uri) => {
+                let _p = profile::span("run_unindexed_project");
+
+                tracing::debug!(changes = self.process_changes(), "processes changes");
```

Comment:
Drive-by remark: does this actually work with `features = ["max_level_info"]` etc.?

Reply:
...you know, i don't think it will. i'll have to move it out.

Reply:
removed it, since it was unnecessary.